cmd/lt-deployer/main.go

```diff
@@ -53,7 +53,7 @@ func getConfig(cmd *cobra.Command) (*deployment.Config, error) {
 
 func main() {
 	rootCmd := &cobra.Command{
-		Use:          "deployer",
+		Use:          "lt-deployer",
```

Comment:
Since we are here I think we should change `deployer` to something else. Maybe something like:
```
lt-deployer create  -> lt-aws deploy
lt-deployer start   -> lt-aws start
lt-deployer stop    -> lt-aws stop
lt-deployer destroy -> lt-aws destroy
```

Reply:
Sounds good. I'd like it not to be tied to the specific hosting service though.
Time for a `lt-ctl` maybe. :grinning: